From: Christian Tismer <tismer@stackless.com>
Date: Fri, 7 Jun 2024 14:57:14 +0200
Subject: Do the transition to Python 3.13

Python 3.13.0rc1 does not stand PyType_Modified of a builtin type.
Since the change is no longer relevant after __signature__
was removed, we can safely ignore this.

Task-number: PYSIDE-2751
Change-Id: I37b447148787e2923a58c091a5c8ac808d579bc0
Reviewed-by: Friedemann Kleint <Friedemann.Kleint@qt.io>
(cherry picked from commit ad18260e583a30505e42b04fd242c52ff36f735c)
---
 build_scripts/config.py                                      | 1 +
 sources/shiboken2/libshiboken/signature/signature_helper.cpp | 1 -
 2 files changed, 1 insertion(+), 1 deletion(-)

diff --git a/build_scripts/config.py b/build_scripts/config.py
index fb27394..0e7f9fd 100644
--- a/build_scripts/config.py
+++ b/build_scripts/config.py
@@ -96,6 +96,7 @@ class Config(object):
             'Programming Language :: Python :: 3.10',
             'Programming Language :: Python :: 3.11',
             'Programming Language :: Python :: 3.12',
+            'Programming Language :: Python :: 3.13',
         ]
 
         self.setup_script_dir = None
diff --git a/sources/shiboken2/libshiboken/signature/signature_helper.cpp b/sources/shiboken2/libshiboken/signature/signature_helper.cpp
index 05eaa14..8ae048b 100644
--- a/sources/shiboken2/libshiboken/signature/signature_helper.cpp
+++ b/sources/shiboken2/libshiboken/signature/signature_helper.cpp
@@ -124,7 +124,6 @@ int add_more_getsets(PyTypeObject *type, PyGetSetDef *gsp, PyObject **doc_descr)
         if (PyDict_SetItemString(dict, gsp->name, descr) < 0)
             return -1;
     }
-    PyType_Modified(type);
     return 0;
 }
 
